modules/develop/pages/consume-data/consumer-offsets.adoc

```diff
@@ -4,20 +4,125 @@
 // tag::single-source[]
 :description: pass:q[Redpanda uses an internal topic, `__consumer_offsets`, to store committed offsets from each Kafka consumer that is attached to Redpanda.]
 
-Redpanda supports `pass:[__consumer_offsets]`, which is a private topic on a Redpanda broker. The `pass:[__consumer_offsets]` topic stores committed offsets from each Kafka consumer that is attached to Redpanda.
+== Overview
```

Comment:
Most of our doc pages don't begin with an **Overview** heading, we just jump right into the introductory text.

Reply:
Agree, but we do have other topics like this one, which have a longish intro/overview, and I think it would simply help frame the introduction more clearly.